paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -55,10 +57,18 @@ def add_subparser(subparsers):
             required=True,
         ).completer = lazy_choices_completer(list_instances)
         status_parser.add_argument(
-            '-c', '--cluster',
-            help='The PaaSTA cluster that has the service you want to %s. Like norcal-prod' % lower,
-            required=True,
-        ).completer = lazy_choices_completer(utils.list_clusters)
+            '-c', '--clusters',
+            choices=list_clusters(),
+            help="A comma-separated list of clusters to view. Defaults to view all clusters.\n"
+            "For example: --clusters norcal-prod,nova-prod"
+        ).completer = lazy_choices_completer(list_clusters)
+        status_parser.add_argument(
+            '--cluster',
+            choices=list_clusters(),
+            help="A single cluster to view.\n"
+            " Deprecated! For example: --cluster norcal-prod."
```

Comment:
We don't need this.

With python args you can use a substring of the arg and it will work.

For example you can say `paasta restart --serv foo`

Just make a "--clusters" arg and it will automatically be reference-able as -- cluster.